### PR TITLE
mt8173: Add ARM-TF source code

### PR DESCRIPTION
**Pull request by m943040028**
Signed-off-by: SY Chiu <sy.chiu@linaro.org>
Tested-by: SY Chiu <sy.chiu@linaro.org> (MT8173 EVB)

### CODE REVIEW

File: mt8173-evb.xml [hunk 1]
```diff
@@ -13,6 +14,9 @@
 	<project path="optee_linuxdriver" name="optee_linuxdriver.git" />
 	<project path="optee_test" name="optee_test.git" />
 
+	<!-- ARM-TF -->
+	<project remote="mtk" path="arm-trusted-firmware" name="arm-trusted-firmware.git" revision="integration" />
```
Comment (pascal-brand-st-dev) [inline mt8173-evb.xml]: Is it possible to have this branch in https://github.com/linaro-swg/arm-trusted-firmware instead of https://github.com/m943040028?

Comment (m943040028) [inline mt8173-evb.xml]: Sure

**Comment by m943040028** [comment]
This PR depends on https://github.com/OP-TEE/build/pull/54

**Comment by jbech-optee-adm** [comment]
Closing this as #23 contained the same fix, but was more up to date.

**Pull request closed without merging**